README.md

```diff
@@ -32,23 +32,39 @@ Laravel Property Bag gives your application resources savable, secure settings b
 composer require zachleigh/laravel-property-bag
 ```
 
-##### 2. Register the service provider        
+#### Laravel Installation
+
+##### a. Register the service provider        
 In Laravel's config/app.php file, add the service provider to the array with the 'providers' key.
 ```
 LaravelPropertyBag\ServiceProvider::class
 ```
 
-##### 3. Publish the migration      
+##### b. Publish the migration      
 ```
 php artisan vendor:publish --provider="LaravelPropertyBag\ServiceProvider"
 ```
 
-##### 4. Run the migration      
+#### Lumen Installation
+
+##### a. Enable Eloquent
+If you're not already did that, find and uncomment `$app->withEloquent()` call in `app/boostrap.php` file.
```

Comment:
Should read: 'If you haven't already done so, find and uncomment `$app->withEloquent()` in `app/boostrap.php`.'